backend/TraderX/src/main/java/cmpe451/group6/authorization/security/JwtTokenProvider.java

```diff
@@ -50,18 +50,25 @@ protected void init() {
 
   public String createToken(String username, List<Role> roles) {
 
+    if(HazelcastService.isWhiteToken(username)){
```

Comment:
System should allow multiple tokens at a time since user might login into website and mobile together or remove the app and install again. This will lock user from login to system. @enozcan

Reply:
Active tokens count is limited by 10 per client on eb6b148629cbfd91c3e9b52210b3df723709c6b2